neural_network_lyapunov/control_affine_system.py

```diff
@@ -113,3 +114,47 @@ def f(self, x):
 
     def G(self, x):
         return self.B
+
+
+def train_control_affine_forward_model(forward_model_f, forward_model_G,
+                                       x_equ, u_equ,
+                                       model_dataset, num_epochs, lr,
+                                       batch_size=50,
+                                       verbose=True):
+    """
+    Helper function to train neural networks that approximate continuous time
+    dynamics as
+    ẋ = ϕ_f(x) + ϕ_G(x) u - ϕ_f(x*) - ϕ_G(x*) u*
+    @param forward_model_f Feedforward network ϕ_f
+    @param forward_model_G Feedforward network ϕ_G
+    @param x_equ Tensor shape (x_dim,) with the system equilibrium state
+    @param u_equ Tensor shape (u_dim,) with the system equilibrium control
+    @param model_dataset TensorDataset with data, label = ([x|u], ẋ)
+    @param num_epochs int number of training epochs
+    @param lr float learning rate
+    @param batch_size int
+    """
+    x_dim = x_equ.shape[0]
+    u_dim = u_equ.shape[0]
+
+    def compute_x_dot(forward_model_f, network_input, forward_model_G=None):
+        assert(forward_model_G is not None)
```

Comment:
forward_model_G is default to NOne in `compute_x_dot`, but we assert it is not None here?

Reply:
somehow I thought that python wouldn't allow regular arguments to be filled as keyword argument (and that's necessary for backward comp of the aprox training function in utils). But looks like python just takes it.